src/main/java/io/confluent/idesidecar/restapi/clients/SchemaErrors.java

```diff
@@ -9,6 +9,35 @@
 import java.time.Duration;
 import java.util.Map;
 import java.util.concurrent.ConcurrentHashMap;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+
+/**
+ * The SchemaErrors class manages schema error caching for different connections.
+ * It uses Caffeine cache to store and retrieve schema errors based on connection and schema IDs.
+ * <p>This class is application-scoped and observes connection lifecycle events to manage the cache.
```

Comment:
The linter should complain about `<p>` not being preceded by a blank line, so this is correct:
```suggestion
 *
 * <p>This class is application-scoped and observes connection lifecycle events to manage the cache.
```